mst/security.md

```diff
@@ -142,6 +143,6 @@ Click 'Save Changes' and see this take effect immediately.
 Limiting IP address inbound access is just one option to improve the security of your Timescale
 Cloud database instance. There are many other types of security measures you should take into
 account when securing your data. To learn more about security options within Timescale Cloud,
-visit the [Timescale Cloud Knowledge Base][timescale-cloud-kb].
+visit the [Managed TimescaleDB Knowledge Base][timescale-mst-kb].
```

Comment:
I don't see the corresponding link for this at the bottom of the file. Also, we won't have the KBase for much longer, might be better to redirect to `/mst/` instead